### ISSUE TITLE
2 Required deps to compile apk are in devDependencies instead of dependencies

### ISSUE THREAD

**Issue by emanuelb** [hsjoberg/blixt-wallet#313]
compiling the app via `cd android; ./gradlew assembleRelease` after running `yarn install --production` (will not install devDependencies) will result in errors unless both deps `metro-react-native-babel-preset` `remote-redux-devtools` are installed with
```shell
yarn add metro-react-native-babel-preset remote-redux-devtools;
```

which are currently in devDependencies:
https://github.com/hsjoberg/blixt-wallet/blob/f71369d806a1460ca7134388920058fab3781f16/package.json#L152
https://github.com/hsjoberg/blixt-wallet/blob/f71369d806a1460ca7134388920058fab3781f16/package.json#L154

also it's worth to test the same process of using `NODE_ENV="production"` env variable when compiling the app when:
1. compiling the ios version.
2. using `yarn install --frozen-lockfile --production` to install deps (the additonal `--frozen-lockfile` with `--production` is like `npm ci install` command, helpful for reproducible-builds, to have same deps installed).

and if building failed with other missing deps that are in devDependencies move them to Dependencies which need to include all deps required to build the project in production/ci mode.

**Comment by hsjoberg** [comment]
Good catch! I'll have an fix out soon.

**Comment by emanuelb** [comment]
another such dep in depDev is `@babel/runtime`, without it `cd android; ./gradlew assembleRelease` return:
> Error: Unable to resolve module @babel/runtime/helpers/interopRequireWildcard from /home/appuser/app/blixt/blixt-wallet/index.js: @babel/runtime/helpers/interopRequireWildcard could not be found within the project or in these directories:

it should also be moved to Dependencies, tested when compiling latest apk for version 2.0

**Comment by hsjoberg** [comment]
Fixed in c62acbc4fa129b234282d6943d96053820bdb0d6
